Products/ZenUI3/browser/resources/js/zenoss/form/ComponentGraphPanel.js

```diff
@@ -130,8 +131,11 @@
                         width: 10
                     }, {
                         xtype: 'checkbox',
-                        fieldLabel: _t('All on Same Graph?'),
-                        labelWidth: 150,
+                        baseCls: 'zencheckbox_allonsame',
+                        fieldLabel: _t('All on same graph'),
+                        labelAlign: 'right',
+                        labelSeparator: '',
+                        labelWidth: 272,
```

Comment:
this label appears to take up the full width available to it which means clicking in the empty space next to the label causes the checkbox to be selected. This usually isnt a big deal, but it can take a while to create the all-in-one graph,  as the result of an accidental click.